Introduction to Kotlin Multiplatform/06_jvm_render.md

```diff
@@ -1,33 +1,39 @@
-# Rendering in JavaScript
+# Common Rendering and JVM
 
 It is always good to question ourselves - do we really need the
 JVM backend for the project or not. What if we would be able
-to do the [Mandelbrot Set](https://en.wikipedia.org/wiki/Mandelbrot_set)
+to run the [Mandelbrot Set](https://en.wikipedia.org/wiki/Mandelbrot_set)
 rendering on the client? 
 
 Let's implement the client-side rendering and see how it goes.
 
 ## Sharing Code
 
 We've just added the very first Kotlin file into the `commonMain`
-source set. Now let's move all files from the `jvmMain` folder
-into the `commonMain` folder to share the code with JavaScript.
+source set. Let's move all the files from the `jvm` source set
+into the `common` source set to share the code with JavaScript.
 
-Wait a moment. It will not work this way because we use JVM APIs
-to render images. On the client-side we should be using
+Wait a moment. It will not work directly this way because we use JVM APIs
+to render images in our code. We need to pick another APIs to use in
+a browser Kotlin/JS part.
+We will be using the
 [HTML Canvas](https://www.w3schools.com/html/html5_canvas.asp)
-instead.
+to render our images in JavaScript.
 
+It is easy to share code between platforms with Kotlin. 
 [Kotlin Multiplatform](https://kotlinlang.org/docs/reference/multiplatform.html)
-introduces `expect` and `actual` keywords to solve that problem. 
+introduces `expect` and `actual` keywords especially for that. 
 We will use the `expect` keyword in the `commonMain` sources specify
 the declarations that we can only implement for a given target.
+The `actual` keyword binds the `expect`-ed declarations with their
+platform specific counterparts.
+
 Let's see how it works.
 
 ## Gradle Dependency
 
 Before we proceed with the code moving, we should include yet another Gradle
```

Comment:
Before we proceed with moving the code, we should include another Gradle
dependency for the `commonMain` source set. Let's add a few more lines
to the `build.gradle.kts` for this:

```kotlin
@@ -44,21 +50,24 @@ common library. It opens the ways to re-use the HTML rendering
code between our JVM server-side and JS client-side.